play

```diff
@@ -63,8 +61,23 @@ JVM_OPTIONS="$JVM_OPTIONS -Dsun.security.ssl.allowLegacyHelloMessages=false"
 JVM_OPTIONS="$JVM_OPTIONS -Dkey.password=$KEY_PASSWORD"
 JVM_OPTIONS="$JVM_OPTIONS -DcertificateDirectory=$DIR/scripts"
 
-AGENT=$DIR/target/universal/stage/jetty-alpn-agent/jetty-alpn-agent-2.0.6.jar
+# ssl-play requires an ALPN Agent. This is downdloaded and stored in the target folder.
+# This snippet detects when the ALPN agent hasn't been downloaded yet and runs
+# `sbt stage` to obtain it.
+export AGENT_TEST=$(pwd)/$(find target -name 'jetty-alpn-agent-*.jar')
+export NUM_AGENTS_FOUND=$(echo $AGENT_TEST| grep target | wc -l)
+
+if [ $NUM_AGENTS_FOUND -eq "0" ]; then
+   echo "No ALPN Agent jar found, running 'sbt stage'"
+   sbt stage;
+fi
 
+# Start `sbt` with the JVM_OPTIONS and the ALPN agent
+export AGENT=$(pwd)/$(find target -name 'jetty-alpn-agent-*.jar' | head -1)
+echo "Detected ALPN Agent: $AGENT "
 
 # Run Play (this should work for both run and start).
-SBT_OPTS=-javaagent:$AGENT $SBT $JVM_OPTIONS "$* $JVM_OPTIONS";
```

Comment:
So does this break either run or start by not passing `"$* $JVM_OPTIONS"` anymore?

Reply:
that script was completely broken. 

$SBT is pointing to `sbt`

So we were setting the SBT_OPTS to:
`-javaagent:agent.jar sbt ...`

And we were not calling `sbt` at the end.

Reply:
You are right. However, the script wasn't working either. I'm don't remember the error I was getting.

Branch `2.6.x` is working, but not after the updates. So, probably related to that. 

In any case, the change proposed on this PR removes the problem with the version of the alpn and it automatically decided if it needs to run `sbt stage`.

Reply:
> In any case, the change proposed on this PR removes the problem with the version of the alpn and it automatically decided if it needs to run `sbt stage`.

That's great.

But does it break either run or start by not passing `"$* $JVM_OPTIONS"` anymore?

Reply:
Not a blocker.